packages/esm-appointments-app/src/patient-chart/patient-appointments-action-menu.component.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Layer, OverflowMenu, OverflowMenuItem } from '@carbon/react';
+import { launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
+import { showModal, useLayoutType } from '@openmrs/esm-framework';
+import type { Appointment } from '../types';
+import styles from './patient-appointments-action-menu.scss';
+
+interface appointmentsActionMenuProps {
+  appointment: Appointment;
+  patientUuid: string;
+}
+
+export const PatientAppointmentsActionMenu = ({ appointment, patientUuid }: appointmentsActionMenuProps) => {
```

Comment:
In general I renamed everything "Patient"* , ie "AppointmentActionMenu" -> "PatientAppointmentActionMenu" and put everything in a "patient-chart" directory.  We can potentially continue to refactor and share widgets between "patient" and "non-patient" contexts going forward, but just wanted to do the basic copying over for now.